dev.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
Can you instead of the notebook add a test or a python example?

Reply:
Added a collection of tests